extensions/blocks/podcast-player/podcast-player.php

```diff
@@ -115,28 +115,42 @@ function render_player( $player_data, $attributes ) {
 		$player_data
 	);
 
+	$secondary_colors  = get_colors( 'secondary', $attributes, 'color' );
+	$background_colors = get_colors( 'background', $attributes, 'background-color' );
+
+	$player_classes_name = trim( "{$secondary_colors['class']} {$background_colors['class']}" );
+	$player_inline_style = trim( "{$secondary_colors['style']} ${background_colors['style']}" );
```

Comment:
Do we actually need to trim those attributes here? Looks to me like `get_colors()` doesn't add unnecessary whitespace

Reply:
if there are not color it will apply an empty string.
```html
<div class=" " style=" " />
```

Reply:
Oh, because `$secondary_colors['style']` or `$secondary_colors['class']` could be empty

Reply:
also both of them